packages/trader/src/Modules/Trading/Components/Elements/Multiplier/cancel-deal-mobile.jsx

```diff
@@ -3,7 +3,7 @@ import { Checkbox, RadioGroup, Dialog, Popover, Text } from '@deriv/components';
 import { localize, Localize } from '@deriv/translations';
 import { connect } from 'Stores/connect';
 import { onToggleCancellation, onChangeCancellationDuration } from 'Stores/Modules/Trading/Helpers/multiplier';
-import Fieldset from 'App/Components/Form/fieldset.jsx';
+import Fieldset from 'App/Components/Form/fieldset.tsx';
```

Comment:
```suggestion
import Fieldset from 'App/Components/Form/fieldset';
```
same applies for rest of the imports in your PR.

Reply:
Thank you, noted